source/localizable/demo/index.html.erb

```diff
@@ -0,0 +1,40 @@
+---
+title:
+  en: Demo request
+  nl: Demo aanvragen
+  de: Demo anfordern
+header_class: header-hero
+---
+
+<div class="hero">
+  <div class="row">
+    <h1><%= t("demo.title") %></h1>
+    <p><%= t("demo.text") %></p>
+
+    <form accept-charset="UTF-8" action="<%= t("demo.form.formkeep") %>" method="POST" class="form-large">
+      <input type="hidden" name="utf8" value="✓">
```

Comment:
Makes it easy to redirect to the right TLD after submitting the form to Formkeep.